26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -20,31 +21,42 @@ Feature: Creator navigates to Description tab
     And they see Save is enabled
 
   Scenario: Creator sees that fields are validated on Description tab
-    Given Creator begins registering a Registration in the Wizard
-    When they navigate to the Description tab
-    And they click the Save button
+    Given Creator begins Wizard registration and navigates to Description tab
+    When they click the Save button
     Then they can see "Mandatory" error messages for fields:
       | Title          |
       | Date published |
 
   @445
   Scenario: Creator searches for Project
-    Given Creator begins registering a Registration in the Wizard
-    When they navigate to the Description tab
+    Given Creator begins Wizard registration and navigates to Description tab
     And they see a Search box for Projects
-    And they enter search term in the Search box
+    When they enter search term in the Search box
     Then they see list of Projects matching the search term
     And they see title and associated Institutions for each Project
 
   @446
   Scenario: Creator adds a Project
-    Given Creator has searched for a Project
-    And they see Search results
+    Given Creator searches for Project
     When they select a Project from the Search results
     Then the selected Project is added to the list of selected Projects
 
   @1774
   Scenario: Creator removes a Project
-    Given Creator has added a Project
+    Given Creator adds a Project
     When they click the Remove Project icon
     Then they see the Project is removed from the list of selected Projects
+
+  @2446
+  Scenario: Creator adds an Allowed Vocabulary
+    Given Creator begins Wizard registration and navigates to Description tab
+    And their Institution has a Vocabulary set as "Allowed"
+    When they click "Add Vocabulary"
+    Then they can add the "Allowed" Vocabulary
+    And they can add values for the added Vocabulary
```

Comment:
er added Vocabulary = "Allowed" Vocabulary? eller er det noen forskjell her? Noen grunn til at `added` ikke er byttet ut med `"Allowed"`?

Reply:
hmm, det mangler kanskje et steg (scenario) her. `added Vocabulary` er et `Allowed Vocabulary` som brukeren _har_ lagt til.

evt kan vi kanskje bare droppe denne setningen, og si at dette scenario er for å legge til et `Allowed Vocabulary`, så har vi eget scenario for å legge til verdier for vocabularies uavhengig av om det er Allowed eller Default. det er på en måte allerede implisitt ved at vi skriver det skal vises i `Scenario: Creator begins Wizard registration and navigates to Description tab` også

Reply:
bedre nå @sarahse ?
splittet opp i to scenario: et for å åpne liste over Allowed Vocabularies, og et for å se inputfelt etter man har valgt det

Reply:
ja, synes det ble bedre 👍